actions/get_formatted_error.py

```diff
@@ -40,8 +40,8 @@ def find_error_execution(self, parent_execution, ignored_error_tasks):
             if (execution.status == "failed" and
                execution.context['orquesta']['task_name'] not in ignored_error_tasks):
                 execution_result = execution.result
-                if execution_result['output']:
-                    if execution_result['output']['errors']:
+                if 'output' in execution_result and execution_result['output']:
+                    if 'errors' in execution_result['output']:
```

Comment:
This is starting to get crazy with nested if's... getting hard to read from an overall function perspective.

One thing you can do here is to flatten the ifs, or make then functions (lots of long / deep functions in this class FYI)

Another thing for this specific line is to do something like: 
```suggestion
                if execution_result.get('output', {}).get('errors')
```

Reply:
Ok, thanks! Addressing now.